app/src/main/java/com/example/imdb/AfterLoginActivity.kt

```diff
@@ -2,23 +2,25 @@ package com.example.imdb
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import androidx.navigation.findNavController
 import androidx.navigation.fragment.NavHostFragment
 import androidx.navigation.ui.setupWithNavController
 import com.example.imdb.databinding.ActivityAfterLoginBinding
 
-class AfterLogin : AppCompatActivity() {
+class AfterLoginActivity : AppCompatActivity() {
+    lateinit var userName: String
     lateinit var binding: ActivityAfterLoginBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityAfterLoginBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
+        val intent = intent
+        userName = intent.getStringExtra("name_user").toString()
 
         val navHostFragment = supportFragmentManager.findFragmentById(R.id.fragmentContainerView) as NavHostFragment
         val navController = navHostFragment.navController
         binding.bottomNavigationView.setupWithNavController(navController)
```

Comment:
I think this is no longer necessary as you're assigning the navigation graph explicitly in the layout

Reply:
that's correct but when I delete those code lines; the navGraph not working. I don't know the why, however I'll make a search. thanks